shared/AppInsightsCommon/src/Interfaces/IConfig.ts

```diff
@@ -118,6 +118,14 @@ export interface IConfig {
      */
     disableFetchTracking?: boolean;
 
+    /**
+     * @description Provide a way to exclude specific route from automatic tracking for XMLHttpRequest or Fetch request.
+     * @type {string[] | RegExp[]}
+     * @memberof IConfig
+     * @defaultValue undefined.
+     */
+     excludeRequestFromAutoTrackingRegex?: string[] | RegExp[];
```

Comment:
Lets be consistent with the other item and call this "Patterns" rather than "Regex"